src/libraries/System.Runtime.Extensions/tests/System/Diagnostics/Stopwatch.cs

```diff
@@ -96,6 +99,25 @@ public static void StartNewAndRestart()
             }
         }
 
+        [Fact]
+        public static void OverridesToString()
+        {
+            // In this test we use string interpolation witch a Stopwatch instance to trigger
```

Comment:
```suggestion
            // In this test we use string interpolation which a Stopwatch instance to trigger
```

Reply:
@tannergooding thanks for catching this, it is actually a typo on my part, I wanted to write:

> ...we use string interpolation **WITH** a Stopwatch instance...

I've pushed another commit with the typo fix 👍